app/controllers/restrooms_controller.rb

```diff
@@ -54,7 +54,17 @@ def update
 
 private
   def list_restrooms
-    @restrooms = Restroom.all.page(params[:page])
+    params[:ada] ||= false
+    params[:unisex] ||= false
+    if params[:unisex] == 'true' && params[:ada] == 'true'
+      @restrooms = Restroom.where('accessible AND unisex').page(params[:page])
```

Comment:
unfortunately, you can't wildcard a boolean lookup (I tried :/) so this code is a bit repetitive